pkg/server/server.go

```diff
@@ -55,6 +55,7 @@ const (
 	devfileEndpoint                  = "/api/devfile/"
 	devfileSamplesEndpoint           = "/api/devfile/samples/"
 	pluginsEndpoint                  = "/api/plugins/"
+	pluginsListEndpoint              = "/api/plugins"
```

Comment:
It looks like we have two endpoints at this path now... We might be overriding an existing endpoint.

Reply:
Or maybe the one just above is a prefix? We should confirm that there's no conflict.

Reply:
@spadgett from what I could tell it seems `/api/plugins/` is a prefix used to request information about individual plugins. The plugin name would come after the last forward slash: https://github.com/openshift/console/blob/master/pkg/plugins/handlers.go#L35

Reply:
I might generalize the endpoint anyway since it's not just plugins. Maybe something like `/api/check-updates` and `updatesEndpoint`?

Reply:
@spadgett done!